module#2/src/scripts/fill-in-db-users.ts

```diff
@@ -0,0 +1,14 @@
+import { sequelize } from "../data-access";
+import { mockUsers } from "../mock";
+import { authenticateMessage } from "../constants";
+import { User } from "../models";
+
+const fillInDbUsers = async () => {
+  await sequelize.authenticate();
+  await User.sync({ force: true });
+  await User.bulkCreate(mockUsers);
```

Comment:
Honestly I don't understand. Ok, firs of all I added the first line about `authenticate` to check connection as mentioned in the documentation. The second line is about synchronizing database and model, crate table or drop it. The last line about populating database with mock data. This is what I tried to accomplish. But the first question is hard for me. As far as I know await is `wait` statement while the previous one. Please provide more information or clarification if I have to do something))

Reply:
@Taximu hi again) I changed and code looks something like this:
![image](https://user-images.githubusercontent.com/73526360/211539568-91d474e8-7df5-45e3-a925-aff1c27b9baa.png)
What I think, I leave `await` for `sequelize.authenticate()` because from my opinion I have to block execution and wait till  `authenticate` will be passed. If error will occur, therefore `catch` block will be executing. The next I also leave `await` for `User.sync({ force: true });` operation because I believe I need to wait until I want to populate BD with data. And the last one I removed `await` for `User.bulkCreate(mockUsers);` because we don't need the result of this operation and not necessary to wait. 
Also I changed `uri` for equalize because I changed password to server in `ElephantSQL`
https://github.com/kayarmolenka/nodejs-global-mentoring-program-2022/commit/669472cfa009a9925a073a544491a3b493393610#diff-b0da084c554c86d607a9e5d147233001cf019081697e15347a27e722932b0ea5L4

Also I started learn a lot of information about `eventLoop` in NodeJS, but need some time and I'll proceed with it later.

Reply:
1). Promises vs Async/Await;
2). What is behind Async/Await;
3). When to use Async/Await;